portal/index.php

```diff
@@ -439,7 +439,7 @@ function enableVerifyBtn(){
             </form>
         <?php } else {
             ?> <!-- Main logon -->
-        <img class="img-responsive center-block login-image" src='<?php echo $GLOBALS['images_static_relative']; ?>/login-logo.png' />
+        <img class="img-responsive center-block login-image" src='<?php echo "../sites/" . $_SESSION['site_id'] . "/images"; ?>/login-logo.png' />
```

Comment:
here also need to check for the directory and the comment

Since this in in 2 places, would recommend doing the file check and comment in globals.php script and setting the logo to variable `$GLOBALS['loginlogopath']` or something like that, which then can use in both the core/portal login scripts.